README.md

```diff
@@ -262,6 +262,15 @@ required plugins.
   * Use the "core-modules" profile if working on the Pulsar core modules since the source code for those modules can be compiled in IntelliJ.
   * Sometimes it might help to mark a specific project ignored in IntelliJ Maven UI by right-clicking the project name and select **Ignore Projects** from the menu.
   * Currently, it is not always possible to run unit tests directly from the IDE because of the compilation issues. As a workaround, individual test classes can be run by using the `mvn test -Dtest=TestClassName` command.
+    
+* The above steps have all been performed, but a test still won't run.
+  * In this case, try the following steps:
+    * Close IntelliJ
+    * Run `mvn clean install -DskipTests` on the command line
+    * Reopen IntelliJ
+  * If that still doesn't work:
+    * Verify maven is using a supported version. Currently, the supported version of Maven is 3.6.1
```

Comment:
we should be consistent with the maven enforcer plugin configuration

https://github.com/apache/pulsar/blob/bacac04f564536be9c30481d2cab88512906967e/pom.xml#L1650-L1652

Reply:
Oh, we're requiring users to be running Maven 3.3.9?